ex2_hal/dfgm/equipment_handler/source/dfgm_handler.c

```diff
@@ -349,9 +349,17 @@ void dfgm_rx_task(void *pvParameters) {
     collecting_HK = 0;
     firstPacketFlag = 1;
 
-    char DFGM_raw_file_name[20] = {0};
-    char DFGM_100Hz_file_name[20] = {0};
-    char DFGM_1Hz_file_name[20] = {0};
+    // Change file system directory to dfgm
+    iErr = red_chdir("VOL0:/dfgm");
```

Comment:
Should be checking red_errno here to be sure it doesn't exist, instead of some other error.